extensions/2.0/Khronos/KHR_materials_clearcoat/README.md

```diff
@@ -62,19 +62,22 @@ All implementations should use the same calculations for the BRDF inputs. Implem
 |**clearcoatRoughnessTexture**     | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)             | The clearcoat layer roughness texture. | No                   |
 |**clearcoatNormalTexture**        | [`normalTextureInfo`](/specification/2.0/README.md#reference-normaltextureinfo) | The clearcoat normal map texture.      | No                   |
   
-The clearcoat formula `f_clearcoat` is computed using the specular term from the glTF 2.0 Metallic-Roughness material, defined in [Appendix B](/specification/2.0/README.md#appendix-b-brdf-implementation).  Use F0 equal `0.04`, base color white `1.0, 1.0, 1.0`, metallic value `0.0`, and the clearcoat roughness value defined in this extension as follows:
+The clearcoat formula `f_clearcoat` is computed using the specular term from the glTF 2.0 Metallic-Roughness material, defined in [Appendix B](/specification/2.0/README.md#appendix-b-brdf-implementation).  Use specular F0 equal `0.04`, base color black `0.0, 0.0, 0.0`, metallic value `0.0`, and the clearcoat roughness value defined in this extension as follows:
```

Comment:
Are we looking for this F0 term from our PBR calculations?
F0 = lerp(dieletricSpecular, baseColor.rgb, metallic)

Reply:
The clearcoat is non-metallic, so this extension is using the dielectricSpecular of 0.04.  The baseColor doesn't really factor into it, but in some implementations you must supply a color there to call the PBR functions.

Reply:
Yes, so the lerp would just pick the dielectric color (currently 0.04) - most shaders would probably calculate it just once then access the value.
Maybe it makes sense to keep the terms the same for sake of being consistent?

Reply:
Is this critical?  I'd like us to finish up clearcoat, it's been open since early 2019.

Reply:
I'm not opposed to the idea, but I'm eager to move this proposal out of the "new ideas" phase and onto ratification.  There are several implementations already deployed.

Reply:
Maybe enough with reference to [Appendix B] and:
`Use specular F0 with a metallic value `0.0` and the clearcoat roughness value defined in this extension as follows:`

On a different note I think we need to mark the section as non-normative - to be consistent with the information in Appendix B.

Reply:
I'm not sure if part of the main formula can be non-normative.  But additional PRs are still welcome after Khronos ratification so long as they don't change the Intellectual Property, so, anyone can still feel free to open a new PR for this.

Reply:
I am planning to put [this](https://github.com/KhronosGroup/glTF/pull/1740#issuecomment-577724563) into #1717 (if there is still interest). This separates the normative and non-normative parts. Whether we use the equations from Enterprise PBR or something else (or both) for the non-normative part is up to discussion.

Reply:
Yes, I know Alex at least is still interested in #1717.